pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -501,6 +507,24 @@ func (asw *actualStateOfWorld) MarkDeviceAsUnmounted(
 	return asw.SetDeviceMountState(volumeName, operationexecutor.DeviceNotMounted, "", "", "")
 }
 
+func (asw *actualStateOfWorld) UpdateReconstructedDevicePath(volumeName v1.UniqueVolumeName, devicePath string) {
+	asw.RLock()
+	defer asw.RUnlock()
```

Comment:
read lock for write operation?

Reply:
Fixed and squashed to "Rework volume reconstruction"